tests/support/case.py

```diff
@@ -549,11 +551,12 @@ def run_cp(self, arg_str, with_retcode=False, catch_stderr=False):
                                catch_stderr=catch_stderr,
                                timeout=60)
 
-    def run_call(self, arg_str, with_retcode=False, catch_stderr=False):
+    def run_call(self, arg_str, local=False, with_retcode=False, catch_stderr=False):
```

Comment:
Is there a reason this `local` arg is being adding in the middle of the list instead of the end? If there are calls to `run_call` that are positional in the codebase, we don't want to cause problems here with the API. (I have the same question about the change on line 213 as well.)

Reply:
no reason. I'll fix that thanks :)